core/src/apps/webauthn/fido2.py

```diff
@@ -209,9 +209,9 @@
 _KEY_AGREEMENT_PUBKEY = nist256p1.publickey(_KEY_AGREEMENT_PRIVKEY, False)
 
 # FIDO2 configuration.
-_ALLOW_FIDO2 = const(True)
-_ALLOW_RESIDENT_CREDENTIALS = const(True)
-_ALLOW_WINK = const(False)
```

Comment:
AFAIK const is not needed for `bool` values

Reply:
I added the const to help eliminate dead code from the compiled binary, but then I found that it only works with integers. Of course we could use 0 and 1 instead of bools, but right now it wouldn't bring much and might cause more trouble than it's worth.

Reply:
OK